.gitignore

```diff
@@ -42,4 +42,4 @@ env.bak/
 venv.bak/
 ivy_dev/
 *venv/
-venv*
\ No newline at end of file
+venv*
```

Comment:
Not sure why this is throwing changes, but can you copy the raw text of the `.gitignore` on the Ivy Master Branch and paste it into your branches `.gitignore` and push that to this branch? This should remove differences.

Reply:
https://raw.githubusercontent.com/unifyai/ivy/master/.gitignore

Reply:
Okay, I just copied the raw text of .gitignore from the Ivy master branch and pushed the changes. Now both .gitignore files are exactly the same. 😊